### DISCUSSION TITLE
What does "sketchybar -m" do?

### DISCUSSION THREAD

**Discussion by neutonfoo** [FelixKratz/SketchyBar#293] [Q&A] [answered]
Apologies if I missed this in the documentation, but what does `sketchybar -m --set ..." do? I've seen in come up in a couple of plugins on https://github.com/FelixKratz/SketchyBar/discussions/12.

**Comment by FelixKratz** [comment, marked as answer]
Prior to `sketchybar-v2.3.3` it was mandatory to include the `-m` or `--message` argument before any command (just as it is in yabai). I have made this optional here: https://github.com/FelixKratz/SketchyBar/releases/tag/v2.3.3

I have only chosen to keep the `-m` or `--message` arguments to ensure retro-compatibility, but it can be safely omitted everywhere where it is still used as you can see in the code and I would consider it deprecated because it is not included in the docs anymore:
https://github.com/FelixKratz/SketchyBar/blob/9cca91678f0f695e65ac531a23069fddd3a75e05/src/sketchybar.m#L169-L172

**Reply by neutonfoo** [reply]
Thanks for the clarification!
